src/constants/ClientConstants.js

```diff
@@ -77,108 +78,112 @@ export const ALL_WEB_NETWORK_NODES = {
-    "34.87.150.174:50211": "0.0.26",
-    "13.56.4.96:50211": "0.0.27",
-    "34.125.200.96:50211": "0.0.27",
-    "35.198.220.75:50211": "0.0.28",
-    "18.139.47.5:50211": "0.0.28",
-    "54.74.60.120:50211": "0.0.29",
-    "34.142.71.129:50211": "0.0.29",
-    "80.85.70.197:50211": "0.0.29",
-    "35.234.249.150:50211": "0.0.30",
-    "34.201.177.212:50211": "0.0.30",
-    "217.76.57.165:50211": "0.0.31",
-    "3.77.94.254:50211": "0.0.31",
-    "34.107.78.179:50211": "0.0.31",
-    "34.86.186.151:50211": "0.0.32",
-    "3.20.81.230:50211": "0.0.32",
-    "18.136.65.22:50211": "0.0.33",
-    "34.142.172.228:50211": "0.0.33",
-    "34.16.139.248:50211": "0.0.34",
-    "35.155.212.90:50211": "0.0.34",
+    "34.239.82.6": "0.0.3",
+    //NOSONAR
+    "35.237.200.180": "0.0.3",
+    //NOSONAR
+    "3.130.52.236": "0.0.4",
+    "35.186.191.247": "0.0.4",
+    "3.18.18.254": "0.0.5",
+    "35.192.2.25": "0.0.5",
+    "74.50.117.35": "0.0.5",
+    "23.111.186.250": "0.0.5",
+    "107.155.64.98": "0.0.5",
+    "13.52.108.243": "0.0.6",
+    "35.199.161.108": "0.0.6",
+    "3.114.54.4": "0.0.7",
+    "35.203.82.240": "0.0.7",
+    "35.236.5.219": "0.0.8",
+    "35.183.66.150": "0.0.8",
+    "35.181.158.250": "0.0.9",
+    "35.197.192.225": "0.0.9",
+    "177.154.62.234": "0.0.10",
+    "3.248.27.48": "0.0.10",
+    "35.242.233.154": "0.0.10",
+    "13.53.119.185": "0.0.11",
+    "35.240.118.96": "0.0.11",
+    "35.204.86.32": "0.0.12",
+    "35.177.162.180": "0.0.12",
+    "34.215.192.104": "0.0.13",
+    "35.234.132.107": "0.0.13",
+    "52.8.21.141": "0.0.14",
+    "35.236.2.27": "0.0.14",
+    "35.228.11.53": "0.0.15",
+    "3.121.238.26": "0.0.15",
+    "34.91.181.183": "0.0.16",
+    "18.157.223.230": "0.0.16",
+    "34.86.212.247": "0.0.17",
+    "18.232.251.19": "0.0.17",
+    "141.94.175.187": "0.0.18",
+    "34.89.87.138": "0.0.19",
+    "18.168.4.59": "0.0.19",
+    "34.82.78.255": "0.0.20",
+    "52.39.162.216": "0.0.20",
+    "34.76.140.109": "0.0.21",
+    "13.36.123.209": "0.0.21",
+    "52.78.202.34": "0.0.22",
+    "34.64.141.166": "0.0.22",
+    "3.18.91.176": "0.0.23",
+    "35.232.244.145": "0.0.23",
+    "69.167.169.208": "0.0.23",
+    "34.89.103.38": "0.0.24",
+    "18.135.7.211": "0.0.24",
+    "34.93.112.7": "0.0.25",
+    "13.232.240.207": "0.0.25",
+    "13.228.103.14": "0.0.26",
+    "34.87.150.174": "0.0.26",
+    "13.56.4.96": "0.0.27",
+    "34.125.200.96": "0.0.27",
+    "35.198.220.75": "0.0.28",
+    "18.139.47.5": "0.0.28",
+    "54.74.60.120": "0.0.29",
+    "34.142.71.129": "0.0.29",
+    "80.85.70.197": "0.0.29",
+    "35.234.249.150": "0.0.30",
+    "34.201.177.212": "0.0.30",
+    "217.76.57.165": "0.0.31",
+    "3.77.94.254": "0.0.31",
+    "34.107.78.179": "0.0.31",
+    "34.86.186.151": "0.0.32",
+    "3.20.81.230": "0.0.32",
+    "18.136.65.22": "0.0.33",
+    "34.142.172.228": "0.0.33",
+    "34.16.139.248": "0.0.34",
+    "35.155.212.90": "0.0.34",
     // TESTNET
-    "34.94.106.61:50211": "0.0.3",
```

Comment:
why remove the ports

Reply:
The issue arises from this part of the code:

```
new GrpcServicesError(
   GrpcStatus.Timeout,
   ALL_NETWORK_IPS[this.nodeIp],
),
```

A client requested that we display which node the failure occurs on. Previously, this wasn't working for both TLS and non-TLS scenarios due to differing ports. Now, we are only comparing the IP addresses to address this.